### ISSUE TITLE
changeDate event does not offer previous date.

### ISSUE THREAD

**Issue by morfin** [eternicode/bootstrap-datepicker#1383]
how can I retrieve the previous date ? Normally these kinds of events offer the old value as an event attribute. 

Any idea ?

**Comment by sachindavids** [comment]
Yup!.. I'm stuck with this issue too.. Can anyone offer some help?
